eng/common/scripts/get-markdown-files-from-changed-files.ps1

```diff
@@ -0,0 +1,47 @@
+param (
+    # The root repo we scaned with.
+    [string[]] $RootRepo = "./",
```

Comment:
Are we defaulting the the wrong path here for RepoRoot? Should this be something like `$PSScriptRoot/../../.." or something like that?

Reply:
This one is depending the $workingDirectory in yaml file. We are setting the $(Build.SourceDirectory) for the root repo. I agreed it is better to default to ``$PSScriptRoot/../../.."` in this script.